Show the protocols on the top level of the config section

balloob opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->
## Breaking change
<!--
  If your PR contains a breaking change for existing users, it is important
  to tell them what breaks, how to make it work again and why we did this.
  This piece of text is published with the release notes, so it helps if you
  write it towards our users, not us.
  Note: Remove this section if this PR is NOT a breaking change.
-->


## Proposed change
<!--
  Describe the big picture of your changes here to communicate to the
  maintainers why we should accept this pull request. If it fixes a bug
  or resolves a feature request, be sure to link to that issue or discussion
  in the additional information section.
-->
Was discussing with some people. For an organization that loves open standards, we sure don't promote them enough. Most people don't know that we have these dashboards!

This PR breaks up the config section into 3 parts, with the second part being open protocols. Menu items are only shown when you have the integration of course.

Using placeholder icons for KNX and Insteon.


<img width="567" height="553" alt="image" src="https://github.com/user-attachments/assets/95839ec4-258e-4fea-972e-fe88e52f9cf4" />



## Type of change
<!--
  What type of change does your PR introduce to the Home Assistant frontend?
  NOTE: Please, check only 1! box!
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration
<!--
  Supplying a configuration snippet, makes it easier for a maintainer to test
  your PR.
-->

```yaml

```

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your code.
-->

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

<!--
  Thank you for contributing <3
-->

[docs-repository]: https://github.com/home-assistant/home-assistant.io

balloob committed:
Show the protocols on the top level of the config section

MindFreeze commented on src/panels/config/dashboard/ha-config-dashboard.ts:
```diff
@@ -308,18 +310,21 @@ class HaConfigDashboard extends SubscribeMixin(LitElement) {
+                <ha-config-navigation
+                  .hass=${this.hass}
+                  .narrow=${this.narrow}
+                  .showAdvanced=${this.showAdvanced}
```
`showAdvanced` is actually not used. We can remove it

MindFreeze commented on src/panels/config/dashboard/ha-config-dashboard.ts:
```diff
@@ -308,18 +310,21 @@ class HaConfigDashboard extends SubscribeMixin(LitElement) {
+            isComponentLoaded(this.hass, "cloud")
+          ).map(
+            (categoryPages) => html`
+              <ha-card outlined>
```
This creates a `ha-card` even if `categoryPages` is empty.

MindFreeze reviewed (commented):
I like this but not sure about placing HACS with the protocols. It fits more in the first section IMO

wendevlin commented:
Hello,

We discussed it in the Frontend meeting and we are going to wait on a proper design which maybe also include  a merge of integration and configuration pages and a detailed navigation patter on how to come there.

balloob committed:
Address mind freeze comments

balloob committed:
Address comments Matthias pt1

balloob committed:
Reorder based on ABC and vibes

balloob committed:
Address comments Matthias pt1

balloob committed:
Move KNX up

balloob commented:
For icons, I need SVGs that are a single path and single color. I added the Thread one, but somehow the inner circle of the thread-loop fails to render. Kept KNX/Insteon the same as they are not suitable.

Adjusted texts. It all feels a bit "meh" to be honest, as they all are quite similar.

For the ABC-sorting, I didn't do that yet, because I want to keep smart home standards at the top, they are more important.

I see the order like:

- Open Smart home standards: Matter, Zigbee, Z-Wave, KNX
- Open transports: Bluetooth, Thread
- Proprietary smart home standards: Insteon
- Tags

I guess we can also break it up in 2 sections; one for open standards and one for the rest.
